examples/virtual_machines.py

```diff
@@ -171,7 +171,8 @@
 print("\nPower off")
```

Comment:
Usually for setup we using ("\n\npower off"), Please execute the example file and check if all prints are seen correctly

Reply:
Done.